components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -402,6 +402,11 @@ interface KeyringController {
   // - At least 7 characters
   // - At least one non-alphanumeric character
   IsStrongPassword(string password) => (bool result);
+
+  // Obtains if there's a pending unlock request
+  // Note that there is no need for an API to notify unlocked
+  // because the KeyringControllerObserver Unlocked event can be used for that.
+  GetPendingUnlockRequest() => (bool pending);
```

Comment:
nit: How about `HasPendingUnlockRequest` or `IsUnlockRequestPending`?

Reply:
Fixed in f466d97e0c79518206aab34f1e3793c5cf861930